docker/commitment-task.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:jammy
+
+ARG TARGETARCH
+
+RUN apt-get update &&  rm -rf /var/lib/apt/lists/*
```

Comment:
I think there's no need for this unless we're installing packages with apt in the Dockerfile.